src/relay/package.json

```diff
@@ -18,31 +18,27 @@
     "adeira-fetch-schema": "bin/fetch-schema.js",
     "adeira-relay-compiler": "bin/relay-compiler.js"
   },
-  "scripts": {
-    "test": "yarn eslint src && yarn flow src",
-    "regenerate": "./bin/relay-compiler.js --src=./src --schema=./schema.graphql"
-  },
   "dependencies": {
     "@adeira/fetch": "^2.1.0",
     "@adeira/js": "^2.1.1",
     "@adeira/monorepo-utils": "^0.11.0",
     "@adeira/signed-source": "^2.0.0",
     "@babel/register": "^7.16.7",
     "@babel/runtime": "^7.16.7",
-    "babel-plugin-relay": "^12.0.0",
+    "babel-plugin-relay": "^13.0.0",
     "commander": "^8.3.0",
     "is-ci": "^3.0.1",
-    "react-relay": "^12.0.0",
-    "relay-compiler": "^12.0.0",
-    "relay-config": "^12.0.1",
-    "relay-runtime": "^12.0.0"
+    "react-relay": "^13.0.0",
+    "relay-compiler": "^13.0.0",
+    "relay-config": "0.0.0-main-d92b9e55",
```

Comment:
I will remove Relay Config together with the `fetch-schema.js` bin removal. This config dependency is useless in Relay 13+